src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+
+import { Comment } from '../../Types/Comment';
+
+export const CommentInfo: React.FC<Comment> = ({ name, body }) => {
+  const [isShownCommentsList, setCount] = useState(false);
```

Comment:
```suggestion
  const [isShownCommentsList, setIsShownCommentsList] = useState(false);
```
При використанні useState назва функції, яка міняє стейт, повинна починатись із set і далі назва самої змінної зі стейту